packages/blade/src/components/Input/PhoneNumberInput/__tests__/PhoneNumberInput.e2e.stories.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Do we want to write separate stories for E2E? what if test on stories that we write for documentation? This might also force us to cover more cases in documentation itself

Reply:
We can, but E2Es has very specific stories too, it might make the stories confusing if we just keep these as part of main stories.

Reply:
My concern is there's too many stories to write now ;__; 

What if we keep the stories that we don't want to expose to consumer as Internal?

In most cases the stories we write for consumers would work for testing and for the cases where it doesn't, we can make them internal and then test on them?